sdcm/nemesis.py

```diff
@@ -6213,38 +6066,33 @@ def _random_disrupt(self):
         self.execute_disrupt_method(bound_method)
 
 
-class LimitedChaosMonkey(Nemesis):
-
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.disrupt_methods_list = self.get_list_of_methods_compatible_with_backend(limited=True)
+class LimitedChaosMonkey(SisyphusMonkey):
```

Comment:
why not throwing those  completely, i.e. no need of this class anymore.

we can put in the configure where it's needed  

```yaml
nemesis_class: 'SisyphusMonkey'
nemesis_filter: 'limited'
```

those classes only hold the doc that explain  it, it can be next to the definition of the flag  

when it's 1:1 match class to flag, I think we can get rid of the class

keep in mind those classes were create before we have any filter capabilities from the configuration.

i.e. they are just filters